applications/DEMApplication/test_examples/DEM_run_all_benchmarks_grid.py

```diff
@@ -23,7 +23,7 @@
 if "OMP_NUM_THREADS" in os.environ:
     max_available_threads = int(os.environ['OMP_NUM_THREADS'])
 else:
-    max_available_threads = mp.cpu_count() - 1
+    max_available_threads = mp.cpu_count()
```

Comment:
I wouldn't recommend using all the available cpu's when running the benchmarks as grid. Leaving 1 free cpu allows the user to keep working and using other applications and system processes.

Reply:
Ok. I put '-1' back, but with an extra check that the number of processors is not 0. Thanks @farrufat-cimne !

Reply:
true!